warehouse/integrations/bigquery/bigquery_test.go

```diff
@@ -1095,6 +1097,71 @@ func TestIntegration(t *testing.T) {
 			)
 			require.Equal(t, records, whth.SampleTestRecords())
 		})
+		t.Run("multiple files", func(t *testing.T) {
+			tableName := "multiple_files_test_table"
+			repeat := 10
+			loadObjectFolder := "rudder-warehouse-load-objects"
+			sourceID := "test_source_id"
+
+			prefixes := []string{loadObjectFolder, tableName, sourceID, uuid.New().String() + "-" + tableName}
+
+			loadFiles := lo.RepeatBy(repeat, func(int) whutils.LoadFile {
```

Comment:
Is it worth creating load files concurrently ?

Reply:
No. It's just tests. Also, it's just 10 files which should happen very quick.